lms/templates/discussion/user_profile.html

```diff
@@ -23,7 +23,7 @@
 <section class="container">
   <div class="course-wrapper">
     <section aria-label="${_('User Profile')}" class="user-profile">
```

Comment:
@ahsan-ul-haq, can you get rid of the aria-label on this section element?  It should be on the nav only, like you have it below.